From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Mon, 5 Jul 2021 19:00:25 +0200
Subject: [PATCH] Remove outdated URL for BLS document

The document was moved to https://systemd.io/BOOT_LOADER_SPECIFICATION/,
update the URL accordingly to point to the current location.

Resolves: rhbz#1926453

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 util/grub.d/10_linux.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub.d/10_linux.in b/util/grub.d/10_linux.in
index c9296154f51..6ee0a2cf3d0 100644
--- a/util/grub.d/10_linux.in
+++ b/util/grub.d/10_linux.in
@@ -96,7 +96,7 @@ cat <<EOF
 #
 # The $bls_parser parses the BootLoaderSpec files stored in /boot/loader/entries and
 # populates the boot menu. Please refer to the Boot Loader Specification documentation
-# for the files format: https://www.freedesktop.org/wiki/Specifications/BootLoaderSpec/.
+# for the files format: https://systemd.io/BOOT_LOADER_SPECIFICATION/.
 
 EOF
 }
